google/cloud/spanner/keys_test.cc

```diff
@@ -200,27 +199,17 @@ TEST(InternalKeySetTest, BuildToProtoTwoKeys) {
   KeySet ks = ksb.Build();
 
   ::google::spanner::v1::KeySet expected;
-  EXPECT_TRUE(::google::protobuf::TextFormat::ParseFromString(
-      R"(
-keys {
-  values {
-    string_value: "foo0"
-  }
-  values {
-    string_value: "bar0"
-  }
-}
-
-keys {
-  values {
-    string_value: "foo1"
-  }
-  values {
-    string_value: "bar1"
-  }
-}
-all: false
-)",
+  EXPECT_TRUE(google::protobuf::TextFormat::ParseFromString(
+      R"pb(
+        keys: {
+          values: { string_value: "foo0" }
+          values: { string_value: "bar0" }
+        }
+        keys: {
+          values: { string_value: "foo1" }
+          values: { string_value: "bar1" }
+        }
+        all: false)pb",
```

Comment:
It looks weird without a newline before the closing `)pb".`

But if that's what it does, so be it.